package.json

```diff
@@ -2,6 +2,7 @@
   "name": "multiaddr",
   "version": "5.0.0",
   "description": "multiaddr implementation (binary + string representation of network addresses)",
+  "lead-maintainer": "Victor Bjelkholm <victorbjelkholm@gmail.com>",
```

Comment:
shouldn't this be `leadMaintainer` to follow the rest of the casing of keys in package.json?

Reply:
Good point. Now I've to change it all xD